test/hist.jl

```diff
@@ -228,7 +223,7 @@ end
 
 @testset "midpoints" begin
     @test StatsBase.midpoints([1, 2, 4]) == [1.5, 3.0]
-    @test StatsBase.midpoints(linspace(0, 1, 5)) == 0.125:0.25:0.875
+    @test StatsBase.midpoints(Compat.range(0, stop = 1, length = 5)) == collect(0.125:0.25:0.875)
```

Comment:
Does this means we no longer return a range? Sounds like a regression.

Reply:
Thanks for catching that.  Reverted in https://github.com/JuliaStats/StatsBase.jl/pull/362/commits/d5d5436e92d31c3f6344b7be1218bbb472a10c19